detail.py

```diff
@@ -0,0 +1,11 @@
+from django.conf import settings
+from django_mako_plus import view_function, jscontext
+from catalog import models as cmod
+
+@view_function
+def process_request(request, p:cmod.Product = None):
+
```

Comment:
seem's like you're missing some functional code here

Reply:
We appreciate your feedback and have a team working on your issue right now. You'll be notified of our progress in the next 24-48 hours.